### PR TITLE
Installation guide: Show system package commands using sphinx-inline-tabs

### PR DESCRIPTION
**Pull request by mkoeppe**
Preview: https://deploy-preview-37056--sagemath.netlify.app/html/en/installation/source#linux-system-package-installation

Same for the SPKG pages in the reference manual. https://deploy-preview-37056--sagemath.netlify.app/html/en/reference/spkg/gfortran#equivalent-system-packages

Same for the git instructions in the developer guide, using 2 tabs for HTTPS and SSH: https://deploy-preview-37056--sagemath.netlify.app/html/en/developer/github#forking-the-sage-repository

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [ ] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### :hourglass: Dependencies

- Depends on #36957 (merged here)

### CODE REVIEW

File: src/doc/en/developer/github.rst [hunk 1]
```diff
@@ -123,7 +123,11 @@ https://github.com/alice/sage.
 
 Next if you don't have a local Git repo of Sage, then start afresh `cloning
 your fork
-<https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository>`_::
+<https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository>`_:
+
+.. tab:: Using HTTPS
+
```
Comment (kwankyu) [inline src/doc/en/developer/github.rst:129]: For uniformity, just `By HTTPS protocol`?

**Review by kwankyu — APPROVED** [review]
Looks very good to me.

**Comment by kwankyu** [comment]
As you are here, may I ask if you know the reason why sage docs failed to be published on recent beta releases?

**Comment by mkoeppe** [comment]
Thanks for the review!

**Comment by mkoeppe** [comment]
> sage docs failed to be published on recent beta releases

Did they? https://deploy-livedoc--sagemath.netlify.app/html/en/reference/index.html is current as of 10.3.beta6

**Comment by mkoeppe** [comment]
(I updated the links in https://github.com/sagemath/sage/wiki today)

**Comment by kwankyu** [comment]
Ah, just wrong link. Thanks!

**Pull request merged**